exercises/practice/spiral-matrix/.meta/example.c

```diff
@@ -0,0 +1,50 @@
+#include <stdlib.h>
+#include "spiral_matrix.h"
+
+spiral_matrix_t *spiral_matrix_create(int size)
+{
+   spiral_matrix_t *spiral_matrix = calloc(1, sizeof(spiral_matrix_t));
```

Comment:
Small nit:
Consider refactoring type and variable names from `spiral_matrix` to `spiral` so that we then have a nice `spiral->matrix` during use?

Reply:
Sure, good idea